src/robot-config.cpp

```diff
@@ -4,83 +4,60 @@ using namespace vex;
 
```

Comment:
I don't think we want this version of robot-config merged into main, it will erase the current configurations for the bot